arkouda/strings.py

```diff
@@ -525,6 +525,50 @@ def is_title(self) -> pdarray:
             generic_msg(cmd="checkChars", args=f"isTitle {self.objtype} {self.entry.name}")
         )
 
+    @typechecked
+    def strip(self, pattern: Union[bytes, str_scalars] = " ") -> Strings:
+        """
+        Returns a new Strings with all leading and trailing spaces (or set of given characters)
+        stripped from the original.
+
+        Returns
+        -------
+        Strings
+            Strings with all leading and trailing spaces (or set of given characters) stripped from
+            the original
+
+        Raises
+        ------
+        RuntimeError
+            Raised if there is a server-side error thrown
+
+        See Also
+        --------
+        Strings.to_upper
+
+        Examples
+        --------
+        >>> strings = ak.array(['Strings ', '  StringS  ', 'StringS   '])
+        >>> strings.strip()
+        >>> strings
+        array(['Strings', 'StringS', 'StringS '])
+
+        >>> strings = ak.array(['Strings 1', '1 StringS  ', '  1StringS  12 '])
+        >>> strings.strip(' 12')
+        >>> strings
+        array(['Strings', 'StringS', 'StringS '])
```

Comment:
the third element in your results here both still have a trailing space, despite space being part of what's being stripped. Shouldn't these results contain no remaining trailing spaces?

Reply:
Revised the examples to all be out of place.